redhat-appstudio/service-provider-integration-operator
# Update controller-runtime to get fix for global log override bug

### What does this PR do?
Update controller-runtime to get fix for global log override bug.



### Screenshot/screencast of this PR
<img width="1182" alt="Знімок екрана 2022-06-30 о 11 28 00" src="https://user-images.githubusercontent.com/1614429/176630774-2d400bee-00e1-4e6f-9444-d1cc041463eb.png">

### What issues does this PR fix or reference?
All logs reported as "spiaccesscheck" controller logs. See more https://issues.redhat.com/browse/SVPI-160
Issue https://github.com/kubernetes-sigs/controller-runtime/issues/1908
Fixed https://github.com/kubernetes-sigs/controller-runtime/pull/1907

### How to test this PR?
- Deploy `quay.io/skabashn/service-provider-integration-operator:main_2022_06_30__10_51_56`
- Create SPIAccessTokenBinding
- Ensure that logs comes from `"controller":"spiaccesstokenbinding"`
- Create SpiAccessCheck
- Ensure that logs comes from `"controller":"spiaccesscheck"`
